test-projects/invocation-error/worker.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+'use strict'
+
+exports.main = function() {
+    throw new Error("broken worker");
```

Comment:
what is the difference between this test case and [build error](https://github.com/adobe/aio-cli-plugin-asset-compute/pull/3/files#diff-2f2b53a8f1bb8bd122b69c7cdacd27ac)?

Reply:
I think `test-projects/build-error` is a confusing name... it actually fails the worker at execution time, not build time. Might be good to have a real build failure (where the `aio app deploy --skip-deploy` that is [run](https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/base-command.js#L81) as part of `run-worker` and `test-worker` fails... will add.

Reply:
Ah, wait, so `test-projects/build-error` is used in this test case which does simulate a build error: https://github.com/adobe/aio-cli-plugin-asset-compute/pull/3/files#diff-374cb7220a2a9538ed9d773d92bd93e3R136-R140
> fails with exit code 3 if the worker does not build (has no manifest)

It simply misses a `manifest.yml` that leads to the build error.

Let me update the worker.js to include "correct" code to avoid any confusion.